### PR TITLE
YAML-ize current TC_ACE_1_1 test

### PR DESCRIPTION
**Pull request by cecille**
Automates ACE 1.1

### CODE REVIEW

File: src/app/tests/suites/certification/Test_TC_ACE_1_1.yaml [hunk 1]
```diff
@@ -13,1461 +13,413 @@
...
+    - label: "Write ACL giving admin privilege on all EP0"
+      command: "writeAttribute"
+      attribute: "ACL"
+      arguments:
+          value: [
+                  {
+                      FabricIndex: 1,
+                      Privilege: 5, # administer
+                      AuthMode: 2, # case
+                      Subjects: [112233],
```
Comment (bzbarsky-apple) [inline src/app/tests/suites/certification/Test_TC_ACE_1_1.yaml:43]: commissionerNodeId?

File: src/app/tests/suites/certification/Test_TC_ACE_1_1.yaml [hunk 2]
```diff
@@ -13,1461 +13,413 @@
...
-          [1660288412.818725][3478:3483] CHIP:DMG:
-          [1660288412.818775][3478:3483] CHIP:DMG:         SuppressResponse = true,
-          [1660288412.818818][3478:3483] CHIP:DMG:         InteractionModelRevision = 1
-          [1660288412.818856][3478:3483] CHIP:DMG: }
-          [1660288412.819053][3478:3483] CHIP:TOO: Endpoint: 0 Cluster: 0x0000_0028 Attribute 0x0000_0002 DataVersion: 3875972866
-          [1660288412.819168][3478:3483] CHIP:TOO:   VendorID: 65521
-          [1660288412.819281][3478:3483] CHIP:EM: Sending Standalone Ack for MessageCounter:162210353 on exchange 21063i
-          [1660288412.819381][3478:3483] CHIP:IN: Prepared secure message 0xffff83ffda58 to 0x0000000000000001 (1)  of type 0x10 and protocolId (0, 0) on exchange 21063i with MessageCounter:118354980.
-      disabled: true
+          "Read the NOC attribute (Node operational credentials - requires
```
Comment (bzbarsky-apple) [inline src/app/tests/suites/certification/Test_TC_ACE_1_1.yaml:50]: Would be nice if the labels said which test step number it is.

File: src/app/tests/suites/certification/Test_TC_ACE_1_1.yaml [hunk 3]
```diff
@@ -13,1461 +13,413 @@
...
+      response:
+          error: UNSUPPORTED_ACCESS
+
+    - label: "Tead the VendorID attribute (Basic - requires view)"
+      cluster: "Basic Information"
+      command: "readAttribute"
+      attribute: "VendorID"
+
+    # NO PRIVILEGE
+    - label: "Write ACL giving admin privilege ACL, operate for rest of EP0"
```
Comment (bzbarsky-apple) [inline src/app/tests/suites/certification/Test_TC_ACE_1_1.yaml:340]: This label does not look right.

File: src/app/tests/suites/certification/Test_TC_ACE_1_1.yaml (hunk 2 above)
Comment (cecille) [reply src/app/tests/suites/certification/Test_TC_ACE_1_1.yaml:50]: added them to this test, but not planning to go add them to all the others :) . If this is something we want, we should document this somewhere for future script writers.

File: src/app/tests/suites/certification/Test_TC_ACE_1_1.yaml (hunk 3 above)
Comment (cecille) [reply src/app/tests/suites/certification/Test_TC_ACE_1_1.yaml:340]: fixed.

File: src/app/tests/suites/certification/Test_TC_ACE_1_1.yaml (hunk 1 above)
Comment (cecille) [reply src/app/tests/suites/certification/Test_TC_ACE_1_1.yaml:43]: changed in all places - this was up before that change went in, which is why it started like this

**Review by bzbarsky-apple — APPROVED** [review]

**Review by mlepage-google — APPROVED** [review]
Thank you for implementing these tests.

**Comment by lisaaaas** [comment]
Test Case ID

[TC-ACE-1.2] Subscriptions
Describe Issue:
Step 3：Expected Outcome is save SubscriptionId as subscription_noc，But my execution result is timeout
image


I am not sure why I operate on node 2 in interactive mode. I would like to know whether there are preconditions for executing test case TC-ACE-1.2

**Comment by cecille** [comment]
The comment above relates to test case TC-ACE-1.2, which is not covered by this PR. Can you please open an issue on the test case repo with logs and repo steps.

**Pull request merged**